scholia/app/templates/base.html

```diff
@@ -142,9 +142,14 @@
 	    // spaces must be replaced by underscores in the file name
 	    imageName = imageName.replaceAll(" ", "_")
 	    var imageNameMd5 = md5(imageName);
-	    var imageURL = "https://upload.wikimedia.org/wikipedia/commons/" 
+	    var imageURL = "https://upload.wikimedia.org/wikipedia/commons/thumb/" 
 	    imageURL += imageNameMd5[0] + "/" + imageNameMd5.slice(0,2) + "/"
-	    imageURL += encodeURIComponent(imageName);
+	    var filetype = imageName.match(/\.(\w+)$/);
+	    if (filetype[1] === "tif" || filetype[1] === "tiff") {
```

Comment:
Should this also match capital letters?

Reply:
ah yes it should be a case insensitive match instead